polaris-react/src/components/InContextLearning/InContextLearning.tsx

```diff
@@ -1,18 +1,19 @@
 import React, {useState} from 'react';
-
-import {Step} from './components';
+import styles from './InContextLearning.scss';
+import {Header, Step} from "./components"
 import {Button} from '../Button';
 
 interface Props {
-    children: React.ReactElement[];
+    children?: React.ReactElement[];
 }
 
 export function InContextLearning({children}: Props) {
-    const hasMultipleSteps = children.length > 0;
+    const hasMultipleSteps = children ? children.length > 0 : false;
```

Comment:
Nit: Let's check for the support for null safe operators within this project. That can cut out a good deal of the bloat behind ternaries. Something like `!!children?.length` could work in a cases like this.